generate_status_quo.py

```diff
@@ -0,0 +1,345 @@
+        for dirName, subdirList, fileList in os.walk(root):
+            ptype = 'scripts/inventory'
+            for fn in fileList:
+                fp = os.path.join(dirName, fn)
+                bn = os.path.basename(fn).replace('.py', '').replace('.ini', '')
+
+                ptopic = None
+                for topic in topics:
+                    if topic in bn or bn in topic:
+                        ptopic = topic
+                        break
+
+                if ptopic is None:
+                    #print(fn)
+                    for idx,x in enumerate(pluginfiles):
+                        if not x[2]:
+                            continue
+                        xbn = os.path.basename(x[-1]).replace('.py', '')
+                        if xbn in bn or bn in xbn:
+                            ptopic = x[2]
+                            break
+
+                #if ptopic is None:
+                #    import epdb; epdb.st()
+
+                pluginfiles.append([ptype, fn, ptopic, fp])
+
+        # match action plugins to modules
+        for idx,x in enumerate(pluginfiles):
+            if x[2]:
+                continue
+            if x[0] != 'action':
+                continue
+            for pf in pluginfiles:
+                if not pf[2]:
+                    continue
+                if os.path.basename(pf[-1]).replace('.py', '') == os.path.basename(x[-1]).replace('.py', '').replace('.ini', ''):
+                    pluginfiles[idx][2] = pf[2]
+                    break
+
+        # fill in topics via synonyms
+        for idx,x in enumerate(pluginfiles):
+            if x[2]:
+                continue
+
+            #print(x)
+            ptopic = None
+            for k,v in self.synonyms.items():
+                #print('k: %s' % k)
+                if k in x[1]:
+                    # hacky topic matching
+                    for topic in topics:
+                        #print('topic: %s' % topic)
+                        if topic in v:
+                            ptopic = topic
+                            break
+                        if '.' in topic:
+                            tparts = topic.split('.')
+                            if tparts[-1] in v:
+                                ptopic = topic
+                                break
+                if ptopic:
+                    pluginfiles[idx][2] = ptopic
+                    break
+
+        # find which modules use orphaned doc fragments
+        for idx,x in enumerate(pluginfiles):
+            if x[2]:
+                continue
+            if x[0] != 'doc_fragments':
+                continue
+            df = os.path.basename(x[-1])
+            df = df.replace('.py', '')
+            cmd = 'find %s -type f | xargs fgrep -iH %s' % (os.path.join(self.checkout_dir, 'lib', 'ansible', 'modules'), df)
+            logger.info(cmd)
+            (rc, so, se) = run_command(cmd)
+            if rc == 0:
```

Comment:
`run_command` is really unnecessary as stdlib already supports everything:
```suggestion
            import contextlib
            with contextlib.suppress(subprocess.CalledProcessError):
                filenames = subprocess.check_output(cmd, shell=True, text=True).split('\n')
```
Same with decoding. `text=True` makes it return unicode right away.